src/main/java/com/example/bookstore/controller/AdminController.java

```diff
@@ -0,0 +1,59 @@
+package com.example.bookstore.controller;
+
+import com.example.bookstore.entity.User;
+import com.example.bookstore.service.UsersService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/admin")
+public class AdminController {
+
+    @Autowired
+    private UsersService usersService;
+
+    @RequestMapping("")
+    public String home() {
+        return "admin/layout";
+    }
+
+    @RequestMapping("/users")
+    public String manageUser(Model model) {
+        List<User> users = usersService.getAll();
+        model.addAttribute("users", users);
+        return "admin/users/users";
+    }
+
+    @GetMapping("/users/update")
```

Comment:
```suggestion
    @GetMapping("/users/{id}/edit")
```
ri cu nha
id truyền từ path variable chứ ko phải là params variable nha